lib/crowbar/client/command/batch/build.rb

```diff
@@ -14,12 +14,30 @@
 # limitations under the License.
 #
 
+require "easy_diff"
+
 module Crowbar
   module Client
     module Command
       module Batch
         class Build < Base
           def request
+            args.easy_merge!(
+              includes: options.includes,
+              excludes: options.excludes
+            )
+
+            args.file = case args.file
+            when "-"
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)